inc/log.class.php

```diff
@@ -79,17 +79,13 @@ static function getTypeName($nb = 0) {
 
 
    function getTabNameForItem(CommonGLPI $item, $withtemplate = 0) {
-
-      if (!$withtemplate) {
-         $nb = 0;
-         if ($_SESSION['glpishow_count_on_tabs']) {
-            $nb = countElementsInTable('glpi_logs',
-                                       ['itemtype' => $item->getType(),
-                                        'items_id' => $item->getID()]);
-         }
-         return self::createTabEntry(self::getTypeName(1), $nb);
+      $nb = 0;
+      if ($_SESSION['glpishow_count_on_tabs']) {
+         $nb = countElementsInTable('glpi_logs',
+                                    ['itemtype' => $item->getType(),
+                                       'items_id' => $item->getID()]);
       }
-      return '';
+      return self::createTabEntry(self::getTypeName(1), $nb);
```

Comment:
Should this be changed in 9.5/bugfixes branch ?

Reply:
as seen together, yes

Reply:
Done in #7784.